### PR TITLE
remove csv-parse

### PR DESCRIPTION
**Pull request by lane-c-wagner**
#292

### CODE REVIEW

**Comment by preston-wagner** [comment]
actually trying to open a csv containing `name,url,username,password
test.com,https://www.amazon.com,"usernamne,test",dfsadfasdfsdfs
example.co,https://angel.co/login,mail@gmail.com,ghfghffdsfdsf` raises the error `TypeError: Cannot read property 'length' of null`

**Comment by lane-c-wagner** [comment]
Are you talking about the test? If so it should be passing now, it had a merge conflict

**Comment by preston-wagner** [comment]
Not the unit test, I tried manually creating a csv with those contents and loading it, and it error-ed out (my experience with unit tests is that, while nice, they're rarely conclusive)

**Comment by lane-c-wagner** [comment]
hmmm it worked for me, can you try again?

File: src/mixins/storage.js [hunk 1]
```diff
@@ -3,8 +3,8 @@ import { remote } from 'electron';
 const dialog = remote.dialog;
 const pjson = require('../../package.json');
 import { authenticate, isLoggedIn, setToken, getVaults, upsertVault, updateUserPassword } from '../lib/CloudClient/CloudClient';
-import assert from '../lib/assert.js';
-import parse from 'csv-parse/lib/sync';
+import assert from '../lib/assert.es6';
```
Comment (preston-wagner) [inline src/mixins/storage.js]: I don't think we want to revert that, do we?

**Pull request merged**